src/app/loadout-builder/LoadoutBuilder.tsx

```diff
@@ -170,6 +176,38 @@ function LoadoutBuilder({
     minimumPower
   );
 
+  // A representation of the current loadout optimizer parameters that can be saved with generated loadouts
+  // TODO: replace some of these individual params with this object
+  const params: LoadoutParameters = useMemo(
+    () => ({
+      statConstraints: _.compact(
+        _.sortBy(Object.entries(statFilters), ([statName]) =>
```

Comment:
Does it matter if these things are sorted? They get rearranged by the users order right?

Reply:
This code is sorting them by the user's order.